application/modules/test/controllers/Test.php

```diff
@@ -45,7 +47,7 @@ public function load_news() {
 		}
 		
```

Comment:
Please remove this extra space please please

Reply:
Done SIR!

Reply:
Please merge it now :>